src/core/fonts.js

```diff
@@ -718,6 +718,10 @@ var Font = (function FontClosure() {
   function int16(b0, b1) {
     return (b0 << 8) + b1;
   }
+  
+  function neg16(value) {
```

Comment:
To me, `neg16` doesn't seem like a very descriptive function name. Can it be improved?